### DISCUSSION TITLE
REST API - Retreive IP address by FHRP Group

### DISCUSSION THREAD

**Discussion by zmaster7** [netbox-community/netbox#9069] [Q&A] [answered]
I am trying to retrieve an IP address via the REST API that is assigned to a specific FHRP Group ID, and it appears this isn't an option using a GET request on the /api/ipam/ip-addresses/ endpoint. I tried using ?assigned_object_type=101&assigned_object_id=1 to test, even though I did not see that option in the documentation, to no avail. There are options for vminterface_id and interface_id only it appears.

Is there something I am missing, or do I need to put in a request for this functionality?

**Comment by jeremystretch** [comment, marked as answer]
You're not missing anything; this was indeed overlooked when FHRP groups were added. A feature request would be much appreciated. Thanks!

**Reply by zmaster7** [reply]
> You're not missing anything; this was indeed overlooked when FHRP groups were added. A feature request would be much appreciated. Thanks!

Ok, thank you. I've created issue #9081 for the feature request.

**Reply by jeremystretch** [reply]
Thanks!
